se.sjoerd.Graphs.json

```diff
@@ -32,8 +32,11 @@
                 {
                     "type" : "git",
                     "url" : "https://github.com/SjoerdB93/Graphs.git",                    
-                    "commit" : "67d11b4266717010ad4579a521927d9acb907e0d"
+                    "commit" : "cce260561b409981052dd5ee1e9eeaba2ba2185c"
```

Comment:
I've added the clipboard behaviour to the changelist as well, so if you could change this to commit 905c4891c2d6284aef13b890e1868deb01195594 that would be great!

Also Flathub prefers to use the tag as well nowadays, on top of the commit. So that would be "tag": "v1.5.0".